scripts/docker-helper-4.x/cht-docker-compose.sh

```diff
@@ -221,8 +226,8 @@ if [[ -n "${2-}" && -n $projectName ]]; then
 		fi
 
 		if [[ -f "${projectName}.env" ]]; then
-			echo "Removing .env file in this directory..." | tr -d '\n'
-			rm ${projectName}.env
+			echo "Backing up .env file in this directory..." | tr -d '\n'
+			mv ${projectName}.env ${projectName}.env.bak
```

Comment:
Neat idea!  you said:

> remove just rename .env to avoid loosing the db key

Can you explain what use case for keeping the key if you've deleted the CouchDB container and all the data in it?

Reply:
you need to migrate the server or archive it (project ends) but you want to be able to restart it with `init`